pkg/backend/httpstate/stack.go

```diff
@@ -49,6 +48,11 @@ type cloudBackendReference struct {
 	project tokens.Name
 	owner   string
 	b       *cloudBackend
+
+	// defaultOrg is the user's default organization, either configured by the user or determined otherwise.
+	// If unset, will assume that there is no default organization configured and fall back to referencing
+	// the user's individual org.
+	defaultOrg string
```

Comment:
In the interest of getting stuff merged these changes around caching defaultOrg on this struct are definitely fine by themselves and could get merged quickly while we work out the rest.